tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -9464,7 +9465,7 @@ TEST_F(ExecutionTest, HelperLaneTestWave) {
 
   bool testPassed = true;
 
-  D3D_SHADER_MODEL TestShaderModels[] = { D3D_SHADER_MODEL_6_0, D3D_SHADER_MODEL_6_5, D3D_SHADER_MODEL_6_6, D3D_SHADER_MODEL_6_7 };
+  D3D_SHADER_MODEL TestShaderModels[] = { D3D_SHADER_MODEL_6_0 };
```

Comment:
Is this the one you meant to modify?

Reply:
Doh! That’s a bad merge on my part when rebasing :(